apps/prairielearn/src/pages/userSettings/userSettings.html.ts

```diff
@@ -145,9 +175,9 @@ export function UserSettings({
               ${accessTokens.length === 0
                 ? html`
                     <li class="list-group-item">
-                      <span class="text-muted"
-                        >You don&apos;t currently have any access tokens</span
-                      >
+                      <span class="text-muted">
+                        You don&apos;t currently have any access tokens.
```

Comment:
```suggestion
                        You don't currently have any access tokens.
```

Reply:
Changed here and in a few other places in d0fb621.